packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -263,25 +263,19 @@ export default function InspectedElementWrapper(_: Props): React.Node {
         <div className={styles.SelectedComponentName}>
           <div
             className={
-              element.isStrictModeNonCompliant
-                ? styles.StrictModeNonCompliantComponent
-                : styles.Component
+              element.isStrictModeNonCompliant ? undefined : styles.Component
```

Comment:
I believe there was a typo in https://github.com/facebook/react/pull/22886

so the right fix would be:
```suggestion
              element.isStrictModeNonCompliant ? styles.StrictModeNonCompliant : styles.Component
```

could you please update and verify that?

Other changes are fine, maybe we should explore supporting this eslint rule for `react-devtools-*` packages.

Reply:
Ok sure, :-)

Reply:
@hoxyq i thought of supporting the same, but the library is an unmaintained, the last update was 4 years ago.? is it fine to use such library ?

Reply:
> 



> 

yes you are correct, it was a small typo!

Reply:
What library are you referring to there? Sorry, didn't get that

Reply:
Oh, you mean eslint plugin. Maybe there is a maintained one, which has the same purpose?

Reply:
ok @hoxyq meanwhile we can merge this one if its okay and let me research if there is something similar :-)